certbot/certbot/_internal/client.py

```diff
@@ -533,7 +533,13 @@ def deploy_certificate(self, domains, privkey_path,
 
         chain_path = None if chain_path is None else os.path.abspath(chain_path)
 
-        msg = ("Unable to install the certificate")
+        display_util.notify("\nDeploying certificate")
+
+        msg = f"Failed to install the certificate (using the {self.config.installer} plugin)."
```

Comment:
In terms of user-facing language, what do you think about "plugin" vs "authenticator"/"installer", or maybe some other options. I'm not sure I actually remember why I used the word "plugin" here (and in another PR).

I don't know how much users actualize these words when they read them, I'd like to hear what your view is on it. 

For me the priority is having the plugin name in the error output so when truncated output gets pasted on the forums, extra interrogation can be avoided.

Reply:
Ooooh GREAT catch. I've definitely run into confusion with "plugin" before, since people usually assume that means a plugin to the *webserver*, not certbot. On the website instructions we seems to avoid using either, just telling people what to do directly or saying "certbot" if we need a noun -- "Run this command to get a certificate and have Certbot edit your Apache configuration automatically to serve it, turning on HTTPS access in a single step."

I think we only need to not confuse users here, rather than actively explain to them how things are working. (The explaining is for helpers, who will know what it means.)

Looking at a couple third-party [installer](https://github.com/bsorahan/certbot-kong/tree/master) [plugins](https://github.com/gboudreau/certbot-heroku/tree/master) on github (which totally weren't there last time I checked, that's so exciting!), they're not all updating webserver configuration files. Well, I guess technically they are since configuration can be a pretty general term, but they don't all call it that anyway.

How about we just say `using -i {self.config.installer}`? That's minimal enough that it won't be actively confusing, tells a helper what they used, and scans grammatically.